petastorm/codecs.py

```diff
@@ -53,6 +57,8 @@ def __init__(self, image_codec='png', quality=80):
         :param image_codec: any format string supported by opencv. e.g. ``png``, ``jpeg``
         :param quality: used when using ``jpeg`` lossy compression
         """
+        assert OPENCV_AVAILABLE, "{} requires opencv-python to be installed"
```

Comment:
Is `{}` in the string meant to be formatted?

Reply:
yes, my bad. just fixed it